vspace/src/main/java/edu/asu/diging/vspace/web/staff/FileController.java

```diff
@@ -0,0 +1,26 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.IVSFile;
+import edu.asu.diging.vspace.core.services.impl.FileManager;
+
+@Controller
+public class FileController {
+
+    @Autowired
+    private FileManager fileManager;
+
+    @RequestMapping(value = "/staff/files/{id}", method = RequestMethod.GET)
```

Comment:
doesn't this page show the edit form? I would make the url `.../edit` and then move this method into the edit controller.